docs/MigrationGuide.md

```diff
@@ -32,7 +32,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   which provide the same interface as the previous one, with only method name changed.
   Especially, when you use 'PeriodicDeploymentManagerProvider', `delegate` should already return `DeploymentManager` wrapped by caching mechanism.
 * [#4131](https://github.com/TouK/nussknacker/pull/4131) `Parameter.defaultValue` now holds `Option[Expression]` instead of `Option[String]`. You have to wrap a `String` with `Expression.spel()`
-
+* [#4179](https://github.com/TouK/nussknacker/pull/4179) Removed status `FailedStateStatus`. Use `ProblemStateStatus` instead.
```

Comment:
Can we just merge this entry with this `Combine statuses Failing, Failed, Error...`? IMO our users won't be interested in such details. WDYT?

Reply:
Yes. Fixed and rebased.